csharp/src/Google.Protobuf.Test/TestProtos/UnittestProto3.cs

```diff
@@ -209,8 +209,8 @@ public enum TestEnumWithDupValue {
     [pbr::OriginalName("FOO1")] Foo1 = 1,
     [pbr::OriginalName("BAR1")] Bar1 = 2,
     [pbr::OriginalName("BAZ")] Baz = 3,
-    [pbr::OriginalName("FOO2")] Foo2 = 1,
-    [pbr::OriginalName("BAR2")] Bar2 = 2,
+    [pbr::OriginalName("")] Foo2 = 1,
+    [pbr::OriginalName("")] Bar2 = 2,
   }
```

Comment:
Rather than losing the information (even if we're not using it at the moment) I think it would be better to add an extra `bool` property to the attribute, e.g.

```csharp
[pbr::OriginalName("FOO1")] Foo1 = 1,
[pbr::OriginalName("BAR1")] Bar1 = 2,
[pbr::OriginalName("FOO2", PreferredAlias = false )] Foo2 = 1,
[pbr::OriginalName("BAR2", PreferredAlias = false)] Bar2 = 2,
```

It feels cleaner in terms of not trying to make one bit of information represent two things. (It may also be useful in terms of people reading the code.)